specification/cognitiveservices/data-plane/TranslatorText/preview/v1.0-preview.1/TranslatorBatch.json

```diff
@@ -981,46 +1111,6 @@
         }
       }
     },
-    "Status": {
```

Comment:
just curious - why delete this? It seems helpful to have the possible status string values defined where there are status properties

Reply:
hmm, good point...

Reply:
thanks for catching this.